src/contours/smallest_angle.cr

```diff
@@ -0,0 +1,13 @@
+class Contours::SmallestAngle
+  setter :origin, :points, :offset_angle
+
+  def initialize(@origin : Point, @points : Array(Point), @offset_angle = 0.0)
+    @offset_angle += 0.00001
```

Comment:
This value was added to make sure I don't select the origin point as the point with the smallest angle. What if I subtract the origin from the list of points?